tests/test_build_latex.py

```diff
@@ -1389,6 +1389,12 @@ def test_latex_labels(app, status, warning):
     assert result.count(r'\label{\detokenize{index:section1}}') == 1
 
 
+@pytest.mark.sphinx('latex', testroot='latex-figure-in-admonition')
+def test_latex_figure_in_admonition(app, status, warning):
+    app.builder.build_all()
+    compile_latex_document(app, 'python.tex')
```

Comment:
Could you check how `.tex` code is generated here? Compiling is too expensive for testing.

Reply:
Yes, I changed it accordingly. The compilation shows the original error. I have changed it now to be a regression. Thanks!